src/formatting.rs

```diff
@@ -61,6 +61,11 @@ impl FormatTemplate {
         Self::format_contains(&self.full, var) || Self::format_contains(&self.short, var)
     }
 
+    pub fn has_tokens(&self) -> bool {
+        (self.full.is_some() && !self.full.as_ref().unwrap().is_empty())
+            || (self.short.is_some() && !self.short.as_ref().unwrap().is_empty())
```

Comment:
I think
```rust
!self.full.as_ref().map(Vec::is_empty).unwrap_or(true)
```
is more "rustic".

Reply:
> is more "rustic".

Yup. Changed.

Reply:
By the way, it's a bit weird that a commenter cannot mark their comments as "resolved"